tests/include/aws_test_tcp.h

```diff
@@ -26,6 +26,16 @@
 #ifndef AWS_TEST_TCP_H
 #define AWS_TEST_TCP_H
 
+/* UDP Echo Server. TODO update address
+ * Update udptestECHO_SERVER_ADDR# and
+ * udptestECHO_PORT with IP address
+ * and port of unencrypted TCP echo server. */
+#define udptestECHO_SERVER_ADDR0         10
+#define udptestECHO_SERVER_ADDR1         184
+#define udptestECHO_SERVER_ADDR2         166
+#define udptestECHO_SERVER_ADDR3         154
+#define udptestECHO_PORT                 ( 8883 )
```

Comment:
I think we will  have to either rename the file to indicate it is the config file for both tcp and udp or move these constants out to a new udp file

Reply:
yes

Reply:
Note: renaming may need bunch of project file updates for various platforms.

Reply:
Adding a new file also needs additional update, but is not as a big amount of work all at once: we should decide while still in development.